dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -5,7 +5,7 @@ class Plc::UserCourseEnrollmentsController < ApplicationController
   def index
     if user_course_enrollment_params[:course]
       course_name = user_course_enrollment_params[:course].tr('-', '_').titleize
-      @user_course_enrollments = [@user_course_enrollments.find_by(user: current_user, plc_course: Plc::Course.find_by(name: course_name))]
+      @user_course_enrollments = [@user_course_enrollments.find_by(user: current_user, plc_course: ::Course.find_by_name(course_name).try(:plc_course))]
```

Comment:
It's a little disconcerting to me that Using the syntax `Course::find_by_name...` worked in tests but not when I actually hit this code running my local server (needed to be `::Course.find_by_name`)